.travis.yml

```diff
@@ -8,6 +8,12 @@ compiler:
   - gcc
   - clang
 
+env:
+  - CMAKE_FLAGS=
+  - CMAKE_FLAGS=-DTINYSPLINE_DOUBLE_PRECISION=NO
+  - CMAKE_FLAGS=-DTINYSPLINE_DISABLE_CXX11_FEATURES=NO
```

Comment:
Similarly to TINYSPLINE_DOUBLE_PRECISION, TINYSPLINE_DISABLE_CXX11_FEATURES, by default, is NO. So, shouldn't this be YES?

Reply:
Fixing now.  😉